pydriller/repository.py

```diff
@@ -204,6 +205,14 @@ def _prep_repo(self, path_repo: str) -> Generator[Git, None, None]:
                 # git directories because of read-only files.
                 # In this case, just ignore the errors.
                 shutil.rmtree(self._tmp_dir.name, ignore_errors=True)
+            except OSError as e:
+                # Another cleanup error happens on Windows with errno 145
+                # Manually remove files
+                # Otherwise, rethrow errors.
+                if sys.platform == "win32" and e.winerror == 145:
+                    shutil.rmtree(self._tmp_dir.name, ignore_errors=True)
```

Comment:
```suggestion
            except (PermissionError, OSError) as e:
                # On Windows there might be cleanup errors.
                # Manually remove files
                shutil.rmtree(self._tmp_dir.name, ignore_errors=True)
```

Reply:
Yes, it is the same as line 207. As for the `PermissionError`, it seems to be unique on Windows for the file handling issue, but the `OSError` might be generic across different platforms. Do you think we need an additional `except` clause to do some sanity check and rethrow the error? In case we have other generic `OSError`?

Reply:
Combining into one except clause also makes sense, considering we shouldn't expect any error here, besides this annoying Windows-specific problem : (

Reply:
Yeah I'm expecting the error to show up only on Windows, and it's one of those 2. So it's fine to merge them 😄